spec/rubocop/cop/style/map_into_array_spec.rb

```diff
@@ -215,6 +311,36 @@
     RUBY
   end
 
+  it 'does not register an offense when pushing anonymously-forwarded block', :ruby31 do
+    expect_no_offenses(<<~RUBY)
+      def foo(&)
+        dest = []
+        src.each { |e| dest.push(&) }
+        do_something
+      end
+    RUBY
+  end
+
+  it 'does not register an offense when pushing anonymously-forwarded restarg', :ruby32 do
+    expect_no_offenses(<<~RUBY)
+      def foo(*)
+        dest = []
+        src.each { |e| dest.push(*) }
+        do_something
+      end
+    RUBY
+  end
```

Comment:
It should also be ignored for anonymously forwarded kwrestarg (`**`)'.

Reply:
Added ✔️